src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -160,7 +160,6 @@ public final void setExcludedClasses(String... excludedClasses) {
     public void setExcludeClassesRegexps(String... from) {
         Arrays.stream(from)
                 .map(CommonUtil::createPattern)
-                .distinct()
```

Comment:
Similar to others, this just saves us processing time. It doesn't matter how many we have of the same string. I would also think this should be somewhat rare unless they are doing something funky in the config.